app/src/main/java/org/pragyan/dalal18/ui/LoginActivity.kt

```diff
@@ -40,6 +40,8 @@ import javax.inject.Inject
 
 class LoginActivity : AppCompatActivity() {
 
+    private lateinit var binding: ActivityLoginBinding
```

Comment:
Implement a Property Delegate to handle the lifecycle of Bindings. Checkout [fest-android](https://github.com/delta/fest-android/)